prime-router/src/main/kotlin/OrganizationService.kt

```diff
@@ -1,16 +1,33 @@
 package gov.cdc.prime.router
 
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.util.*
+
+
 /**
  * An `OrganizationService` represents the agent that the data hub sends reports
  * (minus the credentials used by that agent, of course). It contains information about
  * the specific topic and schema that the sender needs.
+ *
+ * @param name of the service
+ * @param topic defines the set of schemas that can translate to each other
+ * @param schema defines the schema that the org wishes to receive
+ * @param jurisdictionalFilter defines the set of elements and regexs that filter the topic
+ * @param transforms defines the number of transforms to apply to the report before sending
+ * @param batch defines how to batch reports to the org. If null, then send immediatlely
+ * @param format that the org wishes to receive
+ * @param transport that the org wishes to recieve
  */
 data class OrganizationService(
     val name: String,
     val topic: String,
     val schema: String,
     val jurisdictionalFilter: Map<String, String> = emptyMap(),
     val transforms: Map<String, String> = emptyMap(),
+    val batch: Batch? = null,
     val address: String = "",
```

Comment:
Batch is the new set of properties that define the batching for the batch function.